manifests/params.pp

```diff
@@ -16,6 +16,7 @@
   $ssl_build_dir = '/root/ssl-build'
 
   $node_fqdn = $::fqdn
+  $node_cname = []
```

Comment:
Could we just change this to `cname` ? The `node` noemclature was there for pulp nodes and the old node-installer and I think it helps to not use it now.